debian/postinst

```diff
@@ -67,11 +69,34 @@ upgrade_to_status_cache() {
     ua status 2>&1 > /dev/null
 }
 
+is_release_upgrade() {
+    # Return 0 if the package version we upgrade from does not match the
+    # current series (e.g. 14.04)
+    case $PREVIOUS_PKG_VER in
+        *${VERSION_ID}*)  # os-release didn't change from PREVIOUS_PKG_VER
+            return 1;;
+        * ) return 0;; #  os-release changed from PREVIOUS_PKG_VER
+    esac
+}
+
 case "$1" in
     configure)
+      # Special case: legacy precise creds allowed for trusty esm
+      # do-release-upgrade substitutes s/precise/trusty/ in apt source
```

Comment:
Aha, I see. Perhaps `s/$/ so all we need to do is rename the file/` would make this even clearer?